packages/management/addons-manager/bundle/vendir.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: 1.5.0/bundle/config
```

Comment:
there is no version right?

Reply:
yeah version directory is removed per @yharish991 suggestion

Reply:
@ggpaue this should just be `bundle/config`

Reply:
updated this path